### PR TITLE
FR-1406: Added Example data loader notebook

### PR DESCRIPTION
**Pull request by CK-Gea**
# Pull Request Checklist

- [x] Changes follow the [style guide](https://github.com/finbourne/sample-notebooks/blob/master/docs/FINBOURNE%20notebook%20style%20guide.ipynb)
- [ ] Tests pass
- [x] Raised the PR against the `develop` branch

# Description of the PR

Describe the code changes for the reviewers, explain the solution you have provided and how it fixes the issue

### CODE REVIEW

File: examples/example-data/Example Data Load.ipynb [hunk 1]
```diff
@@ -0,0 +1,741 @@
+{
```
Comment (rizwansaeed) [inline examples/example-data/Example Data Load.ipynb:1]: <p>use python naming conventions for variables and function names throughout (snake case)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (rizwansaeed) [inline examples/example-data/Example Data Load.ipynb:1]: <p>use python naming conventions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (rizwansaeed) [inline examples/example-data/Example Data Load.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #17.            resp = PortfoliosApi.get_portfolio(scope=scope,code=portfolio['Code'])
</pre><p>just do the upsert, also do the create and get return the same response?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (rizwansaeed) [inline examples/example-data/Example Data Load.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #33.    for i,port in portfolios.iterrows():
</pre><p>list comprehension</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (rizwansaeed) [inline examples/example-data/Example Data Load.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #29.            resp = PortfolioGroupsApi.get_portfolio_group(scope=scope, code= portfolioGroup['Group Name'])
</pre><p>just create and catch</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (rizwansaeed) [inline examples/example-data/Example Data Load.ipynb:1]: <p>use python naming convenrtion</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (rizwansaeed) [inline examples/example-data/Example Data Load.ipynb:1]: <p>use python naming convention</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (rizwansaeed) [inline examples/example-data/Example Data Load.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #24.                                              transaction_date=pytz.timezone('UTC').localize(trans['transaction_date'].to_pydatetime()),
</pre><p>this and settlement date should be strings (cutlabels)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (rizwansaeed) [inline examples/example-data/Example Data Load.ipynb:1]: <p>use python naming conventions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (rizwansaeed) [inline examples/example-data/Example Data Load.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    for i,quote in quotes.iterrows():
</pre><p> </p><p>make dictionary comprehension</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (rizwansaeed) [inline examples/example-data/Example Data Load.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #10.                                                                     effective_at= pytz.timezone('UTC').localize(quote['effective_at'].to_pydatetime())), 
</pre><p>should be a str</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (rizwansaeed) [inline examples/example-data/Example Data Load.ipynb:1]: <p>python naming convention</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (rizwansaeed) [inline examples/example-data/Example Data Load.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    for i,rate in rates.iterrows():
</pre><p>dictionary comprehension</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (rizwansaeed) [inline examples/example-data/Example Data Load.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #10.                                                                     effective_at= pytz.timezone('UTC').localize(rate['effective_at'].to_pydatetime())), 
</pre><p>should be str</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (rizwansaeed) [inline examples/example-data/Example Data Load.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #13.                                              lineage= "testckg")
</pre><p> </p><p>this should be something more appropriate</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (CK-Gea) [reply examples/example-data/Example Data Load.ipynb:1]: str not done in other notebooks, took reference from there. Does it matter?

Comment (rizwansaeed) [reply examples/example-data/Example Data Load.ipynb:1]: it'll most likely break at some point when someone adds validation on it, we should fix it now

[the remark this answers is not in the document]
Comment (CK-Gea) [reply examples/example-data/Example Data Load.ipynb:1]: added

[the remark this answers is not in the document]
Comment (CK-Gea) [reply examples/example-data/Example Data Load.ipynb:1]: done

Comment (CK-Gea) [reply examples/example-data/Example Data Load.ipynb:1]: done

Comment (CK-Gea) [reply examples/example-data/Example Data Load.ipynb:1]: done

Comment (CK-Gea) [reply examples/example-data/Example Data Load.ipynb:1]: done

Comment (CK-Gea) [reply examples/example-data/Example Data Load.ipynb:1]: removed, added previously for testing

[the remark this answers is not in the document]
Comment (CK-Gea) [reply examples/example-data/Example Data Load.ipynb:1]: done

Comment (CK-Gea) [reply examples/example-data/Example Data Load.ipynb:1]: done

Comment (CK-Gea) [reply examples/example-data/Example Data Load.ipynb:1]: done

File: examples/example-data/Example Data Load.ipynb [hunk 2]
```diff
@@ -0,0 +1,719 @@
...
```
Comment (rizwansaeed) [inline examples/example-data/Example Data Load.ipynb:1]: <p>make this a simple list</p><p> </p><pre class="ql-syntax" spellcheck="false">Attributes
----------
transactions
cutlabels
quotes
valuation
portfolio groups
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (CK-Gea) [reply examples/example-data/Example Data Load.ipynb:1]: <p>Done</p>

**Pull request merged**